tests/test_search_optimizer.c

```
@@ -239,6 +263,25 @@ static int test_config_apply(void)
+    test_restore_globals(1, 2, 3, UINT64_C(55));
+
+    memset(&cfg, 0, sizeof(cfg));
+    cfg.max_iters = 123;
+    cfg.max_evals = 456;
+    cfg.de_population = 40;
+    cfg.de_weight = (real)0.65;
+    cfg.de_crossover = (real)0.85;
+    cfg.de_init_span = (real)1.75;
+    sr_optimizer_config_apply(&cfg);
+
+    CLEED_TEST_ASSERT(sr_de_population == 40);
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)